### PR TITLE
I dye a little bit inside

### PR DESCRIPTION
**Pull request by Yoghurt4C**
Hii~~

I fixie wixied the RecipeArmorDyes fucksy wucksie!!! xpp

I repwaced the `matches` func wiff a more \~streamylined\~ kawaii `matchBetter` method that make the JVM go sooper dooper fast!! xD *starts twerking*

And, I wediwected two cawws in `getCwaftingWesuwt` t-to make the wecipe be fuwwy awawe of the OweDictionyawy?!?1 x3

Can u pwease merge my pwull wequest senpai?!! UwU

image

### CODE REVIEW

**Comment by unicornbloods** [comment]
Can you please talk normal. I can't understand this shit

**Comment by jss2a98aj** [comment]
I'm going to merge this soon, so expect it in the next update.

File: src/main/java/jss/bugtorch/BugTorch.java [hunk 1]
```diff
@@ -17,60 +18,85 @@
 import jss.bugtorch.config.BugTorchConfig;
 
 @Mod(
-		modid = BugTorch.MODID,
```
Comment (Yoghurt4C) [inline src/main/java/jss/bugtorch/BugTorch.java:20]: thank you idea very cool

**Comment by Roadhog360** [comment]
> I'm going to merge this soon, so expect it in the next update.

Last seen 1203 days ago

**Comment by jss2a98aj** [comment]
> > I'm going to merge this soon, so expect it in the next update.
> 
> Last seen 1203 days ago

:trollface:

**Pull request merged**